samples/bluetooth/peripheral_hr_coded/peripheral_hids_mouse/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
Note to self: these should not be here/were not edited? Look into this.

Reply:
It looks like your `samples/bluetooth/peripheral_hids_mouse` folder has either been moved or copied inside the `samples/bluetooth/peripheral_hr_coded` folder, which is normally next to it.